index.js

```diff
@@ -1,4 +1,21 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const routes = require('./routers');
```

Comment:
```suggestion
const routers = require('./routers');
```

Cuidado ao aplicar essa, porque precisamos mudar mais pra baixo onde a variável é utilizada